src/commands/kicks.js

```diff
@@ -0,0 +1,73 @@
+const Discord = require('discord.js');
+const { SlashCommandBuilder } = require('@discordjs/builders');
+const Kicks = require('../models/kicks');
+
+/**
+ *
+ * @param {Discord.CommandInteraction} interaction
+ */
+async function kicksHandler(interaction) {
+	await interaction.deferReply({
+		ephemeral: true
+	});
+
+	const guild = await interaction.guild.fetch();
+	const users = interaction.options.getString('users');
+
+	const userIds = [...new Set(Array.from(users.matchAll(Discord.MessageMentions.USERS_PATTERN), match => match[1]))];
+
+	const kickListEmbed = new Discord.MessageEmbed();
+	kickListEmbed.setTitle('Previous User Kicks');
+	kickListEmbed.setColor(0xdd6c02);
+
+
+	for (const userId of userIds) {
+		const member = await interaction.guild.members.fetch(userId);
+		const user = member.user;
+
+		const { count } = await Kicks.findAndCountAll({
+			where: {
+				user_id: member.id
+			}
+		});
+
+		kickListEmbed.setDescription(`${user.username} previous kicks:`);
+		kickListEmbed.addFields(
+			{
+				name: `${user.username}'s kicks`,
+				value: count.toString()
+			},
+			{
+				name: 'Kicks Left Until Ban',
+				value: Math.max(0, 3 - count).toString()
+			}
+		);

```

Comment:
Rather than just the count, which isnt super useful, this would probably be more useful if it had details ABOUT the kick